packages/p2p/src/components/app.jsx

```diff
@@ -305,6 +306,11 @@ class App extends React.Component {
     };
 
     changeOrderToggle = value => {
+        if (value === 'active') {
+            this.is_active_tab = true;
+        } else {
+            this.is_active_tab = false;
+        }
```

Comment:
```suggestion
        this.is_active_tab = value === 'active';
```

Reply:
You're right. I'll try this, thank you! 👍
I used a ternary operation for this earlier and es lint rejected it. 😢